scylla/src/transport/session.rs

```diff
@@ -726,7 +732,7 @@ impl Session {
         ))
     }
 
-    /// Perform a batch query  
+    /// Perform a batch query
```

Comment:
Please do not remove the two trailing spaces - they mean a newline, like `<br>` in HTML.
Without them after building the documentation it appears as if all text is in a single line, which looks bad,